src/shared/components/mutationMapper/MutationMapperStore.ts

```diff
@@ -382,10 +295,10 @@ export default class MutationMapperStore
     }
 
     @computed get transcriptsByTranscriptId(): {[transcriptId:string]: EnsemblTranscript} {
-        return _.keyBy(this.allTranscripts.result, ((transcript: EnsemblTranscript) => transcript.transcriptId));
+        return _.keyBy(this.allTranscripts.result as EnsemblTranscript[], (transcript  => transcript.transcriptId));
     }
 
-    @computed get mutationsByTranscriptId(): {[transcriptId:string]: Mutations[]} {
+    @computed get mutationsByTranscriptId(): {[transcriptId:string]: Mutation[]} {
```

Comment:
The correct return type for this function is `Mutation[]`, not `Mutations[]`, `Mutations` is a UI component...